package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@heroku-cli/color": "^1.1.14",
     "@heroku-cli/command": "^8.2.6",
+    "@heroku-cli/typescript-api-schema": "false0.0.1",
```

Comment:
this version doesn't look right 😕 

https://github.com/heroku/typescript-api-schema/releases

Reply:
`@heroku-cli/schema`